ntbs-service/Models/Entities/DataQualityBirthCountryAlert.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ntbs_service.Helpers;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Entities
+{
+    public class DataQualityBirthCountryAlert : Alert
+    {
+        public override string Action => "Please review to see if more accurate information available.";
+        public override string ActionLink =>
+            RouteHelper.GetNotificationPath(NotificationId.GetValueOrDefault(), NotificationSubPaths.Overview);
```

Comment:
As noted in your description, my current PR has a nice solution for the action links... but this might get in first. Might be worth, doing a comment on the other review (so if this gets in first, I can do the updating).

Reply:
Atleast what I deem to be a nice solution.